schemas/oic.types-schema.json

```diff
@@ -11,8 +11,7 @@
         "date": {
             "description": "As defined in ISO 8601. The format is [yyyy]-[mm]-[dd].",
             "type": "string",
-            "format": "date-time",
-            "pattern": "^([0-9]{4})-(1[0-2]|0[1-9])-(3[0-1]|2[0-9]|1[0-9]|0[1-9])$"
+            "format": "date-time"
```

Comment:
This does not fit the comment any more. 
And if you remove the comment, why is an OIC specific type definition needed if it is the exact same definition as format.